src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1373,25 +1377,52 @@ namespace WaterToAirHeatPumpSimple {
                         OutTemp = FinalSysSizing(CurSysNum).OutTempAtCoolPeak;
                         rhoair = PsyRhoAirFnPbTdbW(StdBaroPress, MixTemp, MixHumRat, RoutineName);
                         MixEnth = PsyHFnTdbW(MixTemp, MixHumRat);
-                        MixWetBulb = PsyTwbFnTdbWPb(MixTemp, MixHumRat, StdBaroPress, RoutineName);
                         SupEnth = PsyHFnTdbW(SupTemp, SupHumRat);
                         TotalCapCoeff1 = SimpleWatertoAirHP(HPNum).TotalCoolCap1;
                         TotalCapCoeff2 = SimpleWatertoAirHP(HPNum).TotalCoolCap2;
                         TotalCapCoeff3 = SimpleWatertoAirHP(HPNum).TotalCoolCap3;
                         TotalCapCoeff4 = SimpleWatertoAirHP(HPNum).TotalCoolCap4;
                         TotalCapCoeff5 = SimpleWatertoAirHP(HPNum).TotalCoolCap5;
+                        Real64 FanCoolLoad = 0.0;
+                        if (DataFanEnumType > -1 && DataFanIndex > -1) { // add fan heat to coil load
+                            switch (DataFanEnumType) {
+                            case DataAirSystems::structArrayLegacyFanModels: {
+                                FanCoolLoad = Fans::FanDesHeatGain(DataFanIndex, VolFlowRate);
+                                break;
+                            }
+                            case DataAirSystems::objectVectorOOFanSystemModel: {
+                                FanCoolLoad = HVACFan::fanObjs[DataFanIndex]->getFanDesignHeatGain(VolFlowRate);
+                                break;
+                            }
+                            case DataAirSystems::fanModelTypeNotYetSet: {
+                                // do nothing
+                                break;
+                            }
+                            } // end switch
+                            Real64 CpAir = PsyCpAirFnWTdb(MixHumRat, MixTemp);
+                            if (PrimaryAirSystem(CurSysNum).supFanLocation == DataAirSystems::fanPlacement::BlowThru) {
+                                MixTemp += FanCoolLoad / (CpAir * rhoair * VolFlowRate);
+                            } else if (PrimaryAirSystem(CurSysNum).supFanLocation == DataAirSystems::fanPlacement::DrawThru) {
+                                SupTemp -= FanCoolLoad / (CpAir * rhoair * VolFlowRate);
+                            }
+                        }
+                        CoolCapAtPeak = (rhoair * VolFlowRate * (MixEnth - SupEnth)) + FanCoolLoad;
+                        CoolCapAtPeak = max(0.0, CoolCapAtPeak);
```

Comment:
By not adjusting the enthalpy but adding the load here, doesn't this mean one of the reported enthalpies in the coil sizing details report is wrong?

Reply:
Wet-bulb and enthalpy are calculated from T and w.

     c->coilDesEntWetBulb = Psychrometrics::PsyTwbFnTdbWPb(
         c->coilDesEntTemp, c->coilDesEntHumRat, DataEnvironment::StdBaroPress, "ReportCoilSelection::setCoilCoolingCapacity");
     c->coilDesEntEnth = Psychrometrics::PsyHFnTdbW(c->coilDesEntTemp, c->coilDesEntHumRat);
 
     c->coilDesLvgWetBulb = Psychrometrics::PsyTwbFnTdbWPb(
        c->coilDesLvgTemp, c->coilDesLvgHumRat, DataEnvironment::StdBaroPress, "ReportCoilSelection::setCoilCoolingCapacity");
    c->coilDesLvgEnth = Psychrometrics::PsyHFnTdbW(c->coilDesLvgTemp, c->coilDesLvgHumRat);

    c->cpMoistAir = Psychrometrics::PsyCpAirFnWTdb(c->coilDesEntHumRat, 0.5 * (c->coilDesEntTemp + c->coilDesLvgTemp));

Reply:
OK I see.